src/css/legal.scss

```diff
@@ -6,41 +6,70 @@
         text-decoration: none;
     }
 
+    h1 {
+        font-size: 48px;
+        line-height: 52px;
+        letter-spacing: -2px;
+        font-weight: 800;
+    }
+
+    h2 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 32px;
+        line-height: 40px;
+        font-weight: 700;
+        letter-spacing: -1px;
+        margin: 3rem 0 1rem;
+        &:first-child {
+            margin: 0 0 2rem;
+        }
+    }
+
+    h3 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 24px;
+        font-weight: 500;
+        line-height: 30px;
+        letter-spacing: -.5px;
+        margin: 2rem 0 1.5rem;
+    }
+
+    h4 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 16px;
+        font-weight: bold;
+        line-height: 25px;
+        letter-spacing: -.5px;
+        margin: 1rem 0;
+    }
+
+  
     .heading {
-        h2 {
-            color: $accent-text-color;
-            font-family: 'Vitesse', 'Work Sans', sans-serif;
-            font-size: 20px;
-            font-weight: bold;
-            line-height: 34px;
-            margin: 0.5em 0;
+        h3 {
+            margin: 0;
         }
 
-        p:first-of-type {
-            margin-top: 3.5em;
+        .date {
+            font-family: 'Work Sans', sans-serif;
+            font-size: 24px;
+            line-height: 32px;
+            font-weight: 500;
+            letter-spacing: -.5px;
```

Comment:
I do wonder if it makes sense to set `letter-spacing: -.5px` across the site… May be worth the experiment.